### ISSUE TITLE
Is this project dead?

### ISSUE THREAD

**Issue by oswee** [mksarge/redux-first-routing#4]
Looking to use it in shiny bling-bling new LitElement application, but last updates are in 2017 and there is no major version yet.
In what status is this project?
UPDATE:
Some article: [An Introduction to the Redux-First Routing Model](https://www.freecodecamp.org/news/an-introduction-to-the-redux-first-routing-model-98926ebf53cb/)

**Comment by tannerlyons** [comment]
@mksarge if you'd like, I'm willing to maintain

**Comment by tannerlyons** [comment]
@mksarge bump on this if you'd like :)

**Comment by MoralCode** [comment]
I don't think so. I've been using this for a relatively production-grade project for a while and things seem to work just fine (at least for my use). There don't seem to be many updates that are needed (low issues count) so I'd personally attribute the old modification dates on the code to this repo being "done" moreso than it being out of date.
Should this project need a maintainer, I'd also potentially be willing to help out and keep it working

**Comment by MoralCode** [comment]
what a coincidence that as soon as I say that I end up running into issues. Since it seems like there arent many great substitures for the simplicity of this repo that aren't also unmaintained, I have started a [fork of this repo](https://github.com/MoralCode/simple-redux-first-routing) to hopefully provide a place where fixes can be centralized and published. I cant guarantee i'll maintain it forever or be able to keep publishing in every format that this repo made available, but I'll do at least what is needed for my project and will try and be active for at least the near future in case anyone wants to make contributions
